tally_ho/apps/tally/views/reports/staff_performance_metrics.py

```diff
@@ -53,10 +53,13 @@ def approvals_percantage(self, approvals):
         """
         forms_approved = approvals['forms_approved']
         forms_sent_for_review = approvals['forms_sent_for_review']
+        decimal_digits = 2
```

Comment:
Should we make this a param to this function?

Reply:
makes sense, will do

Reply:
Done